File: 프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.js
function solution(l, r) {
    let answer = [];
    
    for(let num=l;num<=r;num++){
        let count=0;
        let strNum = String(num);
        strNum.split("").forEach(digit => {
          if(digit == "5" || digit == "0") count++;
        });
        
        if(count === strNum.length) answer.push(num);
    }
    
    console.log(answer.length === 0);
    
    answer.length === 0 ? answer.push(-1) : null;
  
    return answer;
}

File: 프로그래머스/0/181921. 배열 만들기 2/배열 만들기 2.java
import java.util.*;

class Solution {
    public int[] solution(int l, int r) {
        List<Integer> list = new LinkedList<>();
       
        for(int i=l;i<=r;i++){
            String numStr = String.valueOf(i);
            int strLengthCount = 0;
            for(char digit: numStr.toCharArray()){
                if(digit == '5' || digit == '0'){
                    strLengthCount++;
                }
            }
            
            if(strLengthCount == numStr.length()){
                list.add(i);     
            }
        }
        
        if(list.isEmpty()) list.add(-1);
        
        int[] answer = list.stream().mapToInt(Integer::intValue).toArray();
        return answer;
    }
}